source/index.html.md

```diff
@@ -2348,6 +2376,97 @@ Returns the initialization (immutable) parameters of a given smart contract, rep
 | `method`  | string | Required | `"GetSmartContractInit"`                                                                         |
 | `params`  | string | Required | A smart contract address of 20 bytes. <br> Example: `"fe001824823b12b58708bf24edd94d8b5e1cfcf7"` |
 
+## GetSmartContractSubState
+
+> **Example request:**
+
+```shell
+curl -d '{
+    "id": "1",
+    "jsonrpc": "2.0",
+    "method": "GetSmartContractSubState",
+    "params": ["fe001824823b12b58708bf24edd94d8b5e1cfcf7","admins",[\"0x9bfec715a6bd658fcb62b0f8cc9bfa2ade71434a\""]]
+}' -H "Content-Type: application/json" -X POST "https://api.zilliqa.com/"
+```
+
+```javascript
+const smartContractState = await zilliqa.blockchain.getSmartContractSubState(
```

Comment:
const smartContractSubState

Reply:
[\"0x9bfec715a6bd658fcb62b0f8cc9bfa2ade71434a\""] seems typo here?

Reply:
and there also some change on `java sdk` side:
```java
public class App {
    public static void main(String[] args) throws IOException {
        HttpProvider client = new HttpProvider("https://api.zilliqa.com");
        String smartContractState = client.getSmartContractSubState("fe001824823b12b58708bf24edd94d8b5e1cfcf7");
        System.out.println(smartContractState);
    }
}
```